docs/ref/contrib/postgres/operations.txt

```diff
@@ -192,3 +192,53 @@ database.
 
     The ``CONCURRENTLY`` option is not supported inside a transaction (see
     :ref:`non-atomic migration <non-atomic-migrations>`).
+
+Adding constraints without enforcing validation
+===============================================
+
+.. versionadded:: 3.2
+
+PostgreSQL supports the ``NOT VALID`` option with the ``ADD CONSTRAINT``
+statement to add a constraint to a table without enforcing validation on
+existing rows. This option is useful if you want to skip the potentially
+lengthy scan of of the table to verify if all existing rows satisfy the
+constraint. This operation only supports
+:class:`~django.db.models.CheckConstraint`.
+
+.. class:: AddConstraintNotValid(model_name, constraint)
+
+    Like :class:`~django.db.migrations.operations.AddConstraint`, but
+    avoids scanning through the table to skip the process of validating the
```

Comment:
I keep tripping up on this line. I think it's the word 'to' that doesn't read right for me.

Reply:
Hmm I understand where you're coming from. How about something like "but avoids scanning through the table, thus skipping the process"?

Reply:
Yes that would work. It may be an option to chop the first half of this sentence. Something along the lines of... it's like `addconstraint` but it avoids validating the constraint on existing rows.